packages/server/src/core/router.ts

```diff
@@ -188,25 +190,30 @@ const reservedWords = [
 ];
 
 /**
- *
+ * @internal
+ */
+export type CreateRouterInner<
```

Comment:
Switching to an interface here makes TypeScript emit the named type instead of actually resolving the type and emitting that. This reduces the size of the server declarations by half.